### ISSUE TITLE
MemSQL does not support row value expressions

### ISSUE THREAD

**Issue by knutwannheden** [jOOQ/jOOQ#8610]
MemSQL does not appear to suport row value expressions at all: https://www.memsql.com/forum/t/row-value-expressions/663

The following query does not even parse:

```sql
select 1
from dual
where (1,2) in (select 1, 2 from dual)
```

Should all corresponding `@Support` annotations have to be changed to explicitly exclude MemSQL?

**Comment by lukaseder** [comment]
Please have a look at the existing emulations. Most RVE can be emulated quite easily. Yours, for example:

```sql
select 1
from t
where (a, b) in (select x, y from u)
```

Emulated:

```sql
select 1
from t
where exists (
  select 1
  from u
  where t.a = u.x
  and t.b = u.y
)
```

Have a look at `RowSubqueryCondition`

**Comment by lukaseder** [comment]
What's going on? 🤯

![image](https://user-images.githubusercontent.com/734593/57384324-ad77c080-71b0-11e9-857c-0aba40e73e7d.png)

**Comment by knutwannheden** [comment]
Right, that helped. I still have some failing RVE tests I need to look into.

**Comment by lukaseder** [comment]
I think there's a regression for all DBs related to quantified comparison predicates. Would have to see the failing tests...

**Comment by knutwannheden** [comment]
I will try to set up another test DB which doesn't support RVEs natively and compare the test results.

I will also close this issue after this.
